customTests/custom-test-setup.sh

```diff
@@ -80,14 +80,10 @@ if lspci | grep -iq NVIDIA ; then
 	TOPO_PATH=/opt/azurehpc/test/azurehpc-health-checks/customTests/topofiles
 
 	mkdir -p $TOPO_PATH
+	cp $SRC_DIR/topofiles/* $TOPO_PATH
 
-	if [ ! -e "$TOPO_PATH/ndv4-topo.xml" ]; then
-		cp $SRC_DIR/topofiles/ndv4-topo.xml $TOPO_PATH
-	fi
+	install_perf_test "cuda"
```

Comment:
where does this `cuda` get consumed in function `install_perf_test`?

Reply:
Thanks Jie, it was left over from the perf test change I made. let me address.